### ISSUE TITLE
Exampel for bad bicycle routing in Bad Hindelang (Germany)

### ISSUE THREAD

**Issue by bkuisle** [graphhopper/graphhopper#553]
Hi,
I'm very new here an I hope I'm right!
I'm a very heavy user of graphhopper as a (bicycle) biker and mostly very satisfied about this router. 
But now I have planned a route for next weekend and found this:

https://graphhopper.com/maps/?point=47.505802%2C10.367285&point=47.506128%2C10.368208&locale=de-DE&vehicle=bike&weighting=fastest&elevation=true&layer=Omniscale

All streets are residential.

Regards
Bernhard Kuisle (Alias OSM b*k*)

**Comment by karussell** [comment]
Thanks (again :)) and the reason could be some problems with elevation see #384 and #517 and the discussion [here](https://discuss.graphhopper.com/t/current-configuration-of-online-demo/218) where a user of GH tries to improve elevation. But this is important to get fixed!

**Comment by ratrun** [comment]
This is the same thing as analysed in #330 and the reason for https://discuss.graphhopper.com/t/elevationweighting-implementation-and-bike2/213. Even if you have 100% right elevation data there always will be findings like this.
@bkuisle: You can try the MTB bike or the racebike profile: These profiles do not weight any elevation data. If you know this segment, please let us know how steep it is in real. According to our data and mathematics we use it is a 10 meters ascend on 82 meters. For such an ascend our calculated speed is about 6km/h. The longer segement has 12 meters ascend. But ss the ascend is much lower, the calculated speed is faster.

**Comment by karussell** [comment]
@bkuisle do you have any real elevation or speed data for us here?

**Comment by bkuisle** [comment]
Hi, thanks for asking.
Yes, I rode it with my bike in Nov 2015 equipped with my ciclomaster bicyle computer to measure the elavations (not absolut, but relatively exact)  and the distances. I startet at the crossing Karl-Haffner-Str. - Marktstraße via Badstraße (as proposed by Graphhopper) to crossing Zillenbachstraße-Schützenstraße and back the direct way.
![graphhopper - routing in hindelang - ciclomaster data](https://cloud.githubusercontent.com/assets/15415692/12446325/a6eaf554-bf69-11e5-90d5-ad94c97d899b.jpg)

Attached are the datas in an screenshot from an excel table.

**Comment by karussell** [comment]
Thanks! So the 10 meters are okay but we still do something wrong here. E.g. when looking at the current picked path there is also a steep increase [here](https://graphhopper.com/maps/?point=47.505751%2C10.366427&point=47.506084%2C10.366078&locale=de-DE&vehicle=bike&weighting=fastest&elevation=true&layer=Omniscale) Not sure if the 1m less is the reason this is preferred over [this expected](https://graphhopper.com/maps/?point=47.50578%2C10.367167&point=47.505983%2C10.367543&locale=de-DE&vehicle=bike&weighting=fastest&elevation=true&layer=Omniscale).

**Comment by marcows** [comment]
Is this related? Wrong routing only with vehicle=bike. All others, including mtb and racingbike, work correctly. The yellow road is just 1 unsplit way in the concerned area.
![vehicle=bike](https://cloud.githubusercontent.com/assets/6334512/14155713/276514ae-f6c3-11e5-9c03-17fef37d4d53.png)
https://graphhopper.com/maps/?point=49.066275%2C8.804088&point=49.066155%2C8.804491&locale=de&vehicle=bike&weighting=fastest&elevation=true&layer=Omniscale

I accidentally stumbled on this when testing routing through destination highways (Staffelweg has motorcar/motorcycle=destination) with various OSM routers.

**Comment by karussell** [comment]
Yes, this seems to be an elevation issue too. We should probably reduce the impact of the elevation a bit to see if this fixes the issue.
